references.html.haml

```diff
@@ -6,6 +6,16 @@ title: References
   %h2 References
   %p How other community projects are using Infinispan
 
+%a(name="quarkus")
+%h3 WildFly
```

Comment:
Just happen to see this come through. Shouldn't this be `Quarkus` instead?

Reply:
should be yess!!!!